src/screens/GroupInfo/index.js

```diff
@@ -48,22 +49,21 @@ const GroupInfo = () => {
     handleOnNameChange,
     handleOnImageClicked,
     newParticipant,
-    selectedUser,
-    handleCloseSelectUser,
-    openModal,
     alertRemoveUser,
     memberName,
     onLeaveGroup,
     profile,
     channelState,
-    handlePressContact,
     setUsername,
     onReportGroup,
     isAnonymousModalOpen,
     blockModalRef,
+    handleCloseSelectUser,
+    handlePressContact,
+    selectedUser,
+    openModal,
     isFetchingAllowAnonDM
   } = useGroupInfo();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of destructuring properties from `useGroupInfo()` has been changed. While this doesn't affect the functionality, it's important to keep a consistent order for readability and maintainability. If there's no specific reason for the change, consider reverting to the original order.
```diff
-     handleOnNameChange,
-     handleOnImageClicked,
-     newParticipant,
-      alertRemoveUser,
-      memberName,
-      onLeaveGroup,
-      profile,
-      channelState,
-      setUsername,
-      onReportGroup,
-      isAnonymousModalOpen,
-      blockModalRef,
-     handleCloseSelectUser,
-     handlePressContact,
-     selectedUser,
-     openModal,
-      isFetchingAllowAnonDM
+    handleOnNameChange,
+    handleOnImageClicked,
+    newParticipant,
+    selectedUser,
+    handleCloseSelectUser,
+    openModal,
+    alertRemoveUser,
+    memberName,
+    onLeaveGroup,
+    profile,
+    channelState,
+    handlePressContact,
+    setUsername,
+    onReportGroup,
+    isAnonymousModalOpen,
+    blockModalRef,
+    isFetchingAllowAnonDM
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->